### PR TITLE
Changed blur images to suggested one

### LINKED ISSUE #308: Blur images on some parts of website
To update the blur images to a clear one from Flickr

### PR DESCRIPTION
**Pull request by aditya81070**
* Banner image 1
* Banner Image 2
* The FOSSASIA Summit is an annual conference...
* Subscribe to stay informed 
Fixes #308 , #325

### CODE REVIEW

**Comment by aditya81070** [comment]
@fossasia/gci16-maintainers please review the PR.

**Review by abishekvashok — COMMENTED** [review]
Can you remove the blured images that you have replaced?

**Comment by aditya81070** [comment]
@abishekvashok okay I will do that

**Comment by aditya81070** [comment]
@abishekvashok I have removed images.

**Comment by aditya81070** [comment]
This last commit fixes #325

**Comment by aditya81070** [comment]
@fossasia/gci16-maintainers please review PR.

**Comment by mariobehling** [comment]
Please provide a test deployment on your own Github account.

**Review by samagragupta — COMMENTED** [review]
Please provide the issue number and its link .

**Comment by aayusharora** [comment]
Hi @aditya81070 
Thanks for this :)

It seems like you have added four images but have used only three.
Please remove IMG_6615.jpg, if that is not of the use.

**Comment by aditya81070** [comment]
@aayusharora  I just replaced that image with new one.

**Comment by aditya81070** [comment]
@mariobehling  Here is my Github development link: [Fossasia](https://aditya81070.github.io/fossasia.org/).

Here are changes as screenshot: -
screenshot from 2018-09-24 09-37-45
screenshot from 2018-09-24 09-37-52
screenshot from 2018-09-24 09-38-07
screenshot from 2018-09-24 09-38-28
screenshot from 2018-09-24 09-38-50
screenshot from 2018-09-24 09-38-54
screenshot from 2018-09-24 09-39-16

**Review by samagragupta — APPROVED** [review]
LGTM

**Comment by aditya81070** [comment]
@mariobehling @aayusharora  please see the pull request. if it looks good please merge it.

**Comment by aditya81070** [comment]
@mariobehling  @aayusharora please review this pull  request.

File: about/index.html [hunk 1]
```diff
@@ -1,700 +1,778 @@
 <!doctype html>
 <html lang="en">
-    <head>
-        <meta charset="utf-8">
```
Comment (abishekvashok) [inline about/index.html:4]: Please don't change the intendation

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: about/index.html (hunk 1 above)
Comment (aditya81070) [reply about/index.html:4]: This was due to autosave beautify. Should I remove these indentation?

**Pull request merged**